lib/pharos/host/debian/scripts/configure-cri-o.sh

```diff
@@ -70,4 +71,9 @@ else
     if [ "$orig_config" != "$(cat /etc/crio/crio.conf)" ]; then
         reload_systemd_daemon "crio"
     fi
+
+    if [ "$orig_version" != "$(/usr/local/bin/crio -v)" ]; then
```

Comment:
Should this be combined with the condition above? Now it will reload it twice if both are true.

Reply:
Right, other OS's exit after reload... will fix.